cirq-google/cirq_google/ops/common_gate_families.py

```diff
@@ -0,0 +1,362 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Common Gate Families used in cirq-google"""
```

Comment:
I might suggest renaming this module `fsim_gate_family.py` unless you have specific ideas about other common gate families that should be included here. Given that this module will not typically be imported directly but instead the `FSimGateFamily` will be imported from `cirq_google` (or maybe `cirq_google.ops`), it seems like having separate modules for each gate family would be fine.

Reply:
done